Buildings/Experimental/OpenBuildingControl/CDL/SetPoints/package.mo

```diff
@@ -0,0 +1,11 @@
+within Buildings.Experimental.OpenBuildingControl.CDL;
+package SetPoints "Package with models for control set points"
+  extends Modelica.Icons.VariantsPackage;
+
+annotation (preferredView="info", Documentation(info="<html>
+This package contains components models to compute set points of control systems.
+For additional models, see also
+<a href=\"modelica://Modelica.Blocks.Continuous\">
+Modelica.Blocks.Continuous</a>.
+</html>"));
```

Comment:
Please remove `For additional models....` as we only want to use `CDL` models when expressing sequences.